backend/controller/ingress/testdata/java/httpingress/src/main/java/xyz/block/ftl/java/test/http/TestHTTP.java

```diff
@@ -0,0 +1,124 @@
+package xyz.block.ftl.java.test.http;
+
+import java.util.List;
+
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.PUT;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+
+import jakarta.ws.rs.core.MediaType;
+import org.jboss.resteasy.reactive.ResponseHeader;
+import org.jboss.resteasy.reactive.ResponseStatus;
+import org.jboss.resteasy.reactive.RestPath;
```

Comment:
Are these types extensions to the JAX-RS standard?

Reply:
Yes. The Quarkus team wanted to try and push similar into the standard but I am not sure where that effort got to.

I will make sure to cover this when we talk about it in Sydney.